custom-recipes/ml-assisted-labeling-deephub-export/recipe.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import dataiku
+from dataiku.customrecipe import get_recipe_config, get_input_names_for_role, get_output_names_for_role
+
+
+def format_labeling_plugin_annotations(image_annotations):
+    """ Format labeling annotations to be compatible with dss object detection (deephub)"""
+
+    deephub_image_annotations = []
+    # if images were skipped during labeling plugin, their annotations is nan:
+    image_annotations = json.loads(image_annotations) if isinstance(image_annotations, str) else []
```

Comment:
This will fail badly if the selected column is not the good one (for instance, if the value is not a json)

![image](https://user-images.githubusercontent.com/5690334/141273749-01fe8734-3ade-44de-b10a-24e0578e082b.png)


probably need to wrap that around a try catch to throw an understandable error ?